lib/active_storage/engine.rb

```diff
@@ -16,10 +16,11 @@ class Engine < Rails::Engine # :nodoc:
 
     initializer "active_storage.routes" do
       require "active_storage/disk_controller"
+      require "active_storage/direct_uploads_controller"
 
       config.after_initialize do |app|
         app.routes.prepend do
-          get "/rails/blobs/:encoded_key/*filename" => "active_storage/disk#show", as: :rails_disk_blob
+          eval(File.read(File.expand_path("../routes.rb", __FILE__)))
```

Comment:
This is an engine so if we create a file `config/routes.rb` inside the gem it will be loaded without us having to add new initializer. Same with controllers. If the controllers are inside `app/controllers` they will be automatically required.